faculty/cli/client.py

```diff
@@ -0,0 +1,169 @@
+"""Interact with a Faculty service."""
+
+# Copyright 2016-2018 ASI Data Science
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from contextlib import contextmanager
+
+import requests
+import faculty.cli.auth
```

Comment:
We usually put an empty line between third party (`click`) and package-internal (`faculty.config`) imports.